frontend/packages/dev-console/src/utils/pipeline-filter-reducer.ts

```diff
@@ -46,3 +46,15 @@ export const pipelineRunStatusFilter = (phases, pipeline) => {
   const status = pipelineRunFilterReducer(pipeline);
   return phases.selected.has(status) || !_.includes(phases.all, status);
 };
+
+export const pipelineResourceFilterReducer = (pipelineResource): string => {
```

Comment:
Was going to suggest types here, but there is a circular dependency between these files 🤦  Okay. Part of the tech debt.